docs/spec_draft.md

```diff
@@ -312,9 +301,9 @@ the IEEE-754 specification. For boolean element type, the behavior is same as
 
 ### Semantics
 
-Performs element-wise bitwise AND of two tensors `lhs` and `rhs` of integer
-types and produces a `result` tensor. For boolean tensors, it computes the
-logical operation.
+Performs element-wise bitwise or logical AND of two tensors `lhs` and `rhs` and
```

Comment:
Can we do the same changes for or/xor/not? 

These are one still having multiple examples:
``` 
stablehlo.divide
stablehlo.or
stablehlo.xor
stablehlo.not
stablehlo.exponential
stablehlo.iota
stablehlo.log
stablehlo.logistic
stablehlo.negate
stablehlo.reverse
stablehlo.slice
stablehlo.sort
stablehlo.sqrt
```

Reply:
In this PR, I wanted to limit myself to just abs..constant, because I didn't have the time to go through all the ops (we've already have plenty of ops :)). Let's track under #454.